### PR TITLE
fix: check for required arguments clientId and clientSecret

### LINKED ISSUE #2: Check required parameters clientId and clientSecret
I suggest to check required parameters `opts.clientId` and `opts.clientSecret` in the `OneDrive` constructor and throw if unspecified:

https://github.com/adobe/helix-onedrive-support/blob/38f56719900b8c92e32e0be71e32c0aa742b07f7/src/OneDrive.js#L73-L74

Otherwise, starting up `test/dev/browser.js` and signing in, will fail with an error message mentioning an `undefined` application, which might not be easy to understand.

### PR DESCRIPTION
**Pull request by dominique-pfister**
fix #2

### CODE REVIEW

File: test/onedrive.test.js [hunk 1]
```diff
@@ -18,8 +18,19 @@ const assert = require('assert');
 const OneDrive = require('../src/OneDrive.js');
 
 describe('OneDrive Tests', () => {
+  it('throws when required parameters are not specified.', async () => {
+    try {
+      // eslint-disable-next-line no-unused-vars
+      const drive = new OneDrive({});
+      assert.fail('This should not happen.');
+    } catch (e) {
+      // this is expected
```
Comment (tripodsan) [inline test/onedrive.test.js]: IIRC, `assert.fail()` will throw an exception. so the test passes in anycase. you need to check the error message or use [assert.throws()](https://nodejs.org/docs/latest-v10.x/api/assert.html#assert_assert_throws_fn_error_message)

Comment (dominique-pfister) [reply test/onedrive.test.js]: Thanks, done

File: test/onedrive.test.js [hunk 2]
```diff
@@ -20,9 +20,7 @@ const OneDrive = require('../src/OneDrive.js');
 describe('OneDrive Tests', () => {
   it('throws when required parameters are not specified.', async () => {
     try {
-      // eslint-disable-next-line no-unused-vars
-      const drive = new OneDrive({});
-      assert.fail('This should not happen.');
+      assert.throws(new OneDrive({}));
```
Comment (tripodsan) [inline test/onedrive.test.js]: but now, you don't need the try/catch :-)

Comment (dominique-pfister) [reply test/onedrive.test.js]: Really? If I remove the try/catch, the test fails ;-)

Comment (dominique-pfister) [reply test/onedrive.test.js]: Right, I was missing the `() => new OneDrive({})` part, thanks, @stefan-guggisberg :-D

**Pull request merged**